_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -284,13 +286,8 @@ protected function test__server_port_value() {
 		$http_ports  = array( 80 );
 		$https_ports = array( 80, 443 );
 
-		if ( defined( 'JETPACK_SIGNATURE__HTTP_PORT' ) ) {
```

Comment:
These need to remain configurable constants. They are used for sites behind terminating proxies where the edge is running on a normal port, but the upstream is running on a weird port.

The correct fix is to use the `X-Forwarded-Port` header, but there are some sites that use these constants instead.